src/back/models/Entity.js

```diff
@@ -290,6 +290,18 @@ function Entity(attributeValues, options) {
     this.Entity.specification.name + '" (it has to be a valid uuid)'
   );
 
+  if (this.permissions) {
+    expect(this.permissions).to.be.an.instanceOf(Object);
+    for (var property in this.permissions) {
+      expect(this.permissions[property]).to.have.any.keys('write', 'read');
```

Comment:
Stop changing the task during development!
permissions: {
 'id0': { read: true }, //valid
 'id1': {}, //valid
 'id2': {read: true, abacate: true}, //cleaned, but valid
 'id3': null //invalid
}